### PR TITLE
Basic rest api

### LINKED ISSUE #7: Add basic Rest API with Quarkus
- set up an basic quarkus api with a hello world route

### PR DESCRIPTION
**Pull request by TheMaxens**
added basic REST-API with Quarkus via template from IntelliJ 
/hello is the existing route 

Let me know if something is missing. 

fix #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,62 @@
-# ball-bearings-with-quarkus
+# ball-bearings-with-quarkus Project
```
Comment (philippabele) [inline README.md:1]: There are a lot of conflicts with your existing README.md

Please make sure, that you keep your origin version and add content on how to start the project.

Describe the prerequisits like JDK, gradle, etc. ... It is good practice to give precise information on how to set up the environment for developing.

**Review by philippabele — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheMaxens) [reply README.md:1]: I copied the contribution guide to a new file "contributing.md" and added a link in README.md

The README.md contains information about starting and building the application. 

README.md contains now the section "Prerequisites"

**Comment by philippabele** [comment]
I tried to run your app in dev mode.

There I tried the following instructions:

- set up new wsl2 environment on windows
- installed openjdk 17 with apt
- installed gradle

By running the command./gradlew ... I noticed, that this dir is missing.

May you help me?

I think that a set up part for a dev environment should not contain limitations for a specific IDE. In my case I use VS Code.

**Comment by TheMaxens** [comment]
Sorry, I forgot to push some files.

`./gradlew quarkusDev` should work now 

Could you please try again?

**Comment by philippabele** [comment]
> @philippabele
> 
> Sorry, I forgot to push some files.
> 
> `./gradlew quarkusDev` should work now
> 
> Could you please try again?

@TheMaxens 

`user@wsl2:/mnt/c/code/ball-bearings-with-quarkus$ ./gradlew quarkusDev
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
Caused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain

user@wsl2:/mnt/c/code/ball-bearings-with-quarkus$ ./gradlew build
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
Caused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain`

**Comment by TheMaxens** [comment]
@philippabele 

Can you test it again, please?

**Review by philippabele — APPROVED** [review]

**Pull request merged**